VAMobile/src/components/Nametag/Nametag.tsx

```diff
@@ -39,6 +39,11 @@ export const Nametag: FC = () => {
   serviceHistory.forEach((service) => {
     if (service.honorableServiceIndicator === 'Y') {
       showVeteranStatus = true
+      logAnalyticsEvent(Events.vama_vet_status_shown())
+    } else if (service.honorableServiceIndicator === 'N') {
+      logAnalyticsEvent(Events.vama_vet_status_nStatus())
+    } else if (service.honorableServiceIndicator === 'Z') {
+      logAnalyticsEvent(Events.vama_vet_status_zStatus(service.characterOfDischarge))
```

Comment:
To prevent this from logging multiple times when re-renders happen, this should be in its own `useEffect`. Even then, it could potentially still be logged twice, since the Nametag component gets rendered on the Home and Profile screens. 

To truly only log this once, you'd either need a global place to track whether this has been logged before, and add that result as a condition in these `if` statements, or to move logging this to a different location where it wouldn't get re-rendered, like the Sync screen since we prefetch the service history there. There's also the option of logging this directly in the query function, especially if we were to set the stale time of this query to `Infinity`, because realistically a user's service history isn't likely to change during a session, which means that query function would only be ran once.

Reply:
Also, not sure if this would make it easier to track, but instead of having separate events for logging the different service indicators, we could have one event called `vama_vet_status_indicator` where the indicator is a required parameter, and the character of discharge as an optional parameter